src/pages/Components/FileUploader/code.md

```diff
@@ -2,40 +2,54 @@
 title: 'Code'
 order: 2
 ---
-
 ## Import
 
+Import the settings and the checkbox `scss` files.
+
 ```css
 @import 'settings-tools/_all-settings';
-@import '...';
+@import 'components/_c.file-uploader';
 ```
 
 ## Basic usage
+> Natively, to upload files we use an `<input type="file"/>`, but in Mozaic, we will customize it with a `<label>` that will act as a button.
 
-Apply the classes...
+Wrap a input[type="file"] and a label inside a div and apply the following classes :
 
 ```html
-<div class="class">Label</div>
+<div class="mc-fileuploader">
+    <input type="file" aria-label="Click to choose a file on your device" class="mc-fileuploader__input" id="import-file">
+    <label for="import-file" class="mc-fileuploader__label">file uploader</label>  
+</div>
 ```
+<preview path="src/pages/Components/FileUploader/previews/file-uploader"></preview>
 
-<preview path="src/pages/Components/FileUploader/previews/FileUploader"></preview>
+## Behaviors
+### Standard file uploader states :
 
-## Variations
+- `hover`
+- `focus` 
+- `disabled`
 
-use the same variations in the same order as the index page
+<preview path="src/pages/Components/FileUploader/previews/file-uploader-all-states"></preview>
```

Comment:
Quite strange
![image](https://user-images.githubusercontent.com/43854639/63838233-45c2cc80-c97d-11e9-87de-205d3bf85a4b.png)

Maybe you should add titles to identify state and keep the same wording in each button because differents wording create width difference.